### PR TITLE
add second level load more button

### PR DESCRIPTION
**Pull request by irmakyz**

### CODE REVIEW

File: src/elements/GeoTree/GeoTreeItem.vue [hunk 1]
```diff
@@ -302,6 +325,18 @@ export default {
       if (this.isSingleItem) return false
 
       return this.isSingleSelectMode || this.isFolderSelectHidden
+    },
+
+    hasMoreResultsToLoad () {
+      if (!this.hasLoadMoreButton) return false
+
+      return this.pageSize * this.visiblePages < this.category[this.keyForSubcategory].length
```
Comment (Navas23) [inline src/elements/GeoTree/GeoTreeItem.vue:333]: Let's use lodash to get the size

**Review by Navas23 — CHANGES_REQUESTED** [review]
With your solution we are only applying it to 1 sublevel

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@geoblink/design-system",
-  "version": "35.5.0-beta.3",
+  "version": "35.5.0-beta.5",
```
Comment (Navas23) [inline package.json:3]: Why do we need 5 betas to test this? should be straightforward to test it in local

**Review by Navas23 — COMMENTED** [review]
Don't we need to pass the slot to the following geo-tree-item?

File: package.json (hunk 2 above)
Comment (irmakyz) [reply package.json:3]: first 2 betas were used before, so i started from 2, i fixed the sub level issue and then saw your comment so i published another one, that's why it jumped from 3 to 5

File: src/elements/GeoTree/GeoTreeItem.vue [hunk 3]
```diff
@@ -105,6 +107,12 @@
             />
           </template>
         </geo-tree-item>
+        <geo-list-footer-button
+          v-if="hasMoreResultsToLoad"
+          @click="loadNextPage"
+        >
+          <slot name="moreItemResultsTextContent" />
```
Comment (dletran) [inline src/elements/GeoTree/GeoTreeItem.vue:114]: I feel like we're mixing names in several places, I think we should either stick with `results` or `items` so we either have `moreResultsTextContent` or `moreItemsTextContent`

So if the flag is going to be `hasMoreResultsToLoad`, I'd say to use results also, wdyt?

**Comment by Navas23** [comment]
> With your solution we are only applying it to 1 sublevel

Have you checked this? is this what we want?

**Comment by irmakyz** [comment]
> > With your solution we are only applying it to 1 sublevel
> 
> Have you checked this? is this what we want?

yes, i fixed it you can see in the commit

**Comment by irmakyz** [comment]
> Don't we need to pass the slot to the following geo-tree-item?

yes, i passed it after your comments on tuesday

File: src/elements/GeoTree/GeoTreeItem.vue (hunk 3 above)
Comment (dletran) [reply src/elements/GeoTree/GeoTreeItem.vue:114]: as I mentioned I'd only use results or items but not both, so if we want to tell them apart let's use `moreItemsTextContent`

Comment (irmakyz) [reply src/elements/GeoTree/GeoTreeItem.vue:114]: okay then i will use moreItemsTextContent and moreCategoriesTextContent
is it better?

Comment (irmakyz) [reply src/elements/GeoTree/GeoTreeItem.vue]: - [ ] rename text contents

**Review by dletran — APPROVED** [review]

**Review by fofi — APPROVED** [review]

**Review by Navas23 — APPROVED** [review]

**Pull request merged**